.github/workflows/tests.yml

```diff
@@ -13,19 +13,17 @@ concurrency:
 jobs:
   tests:
     if: github.repository_owner == 'Qiskit'
-    name: macOS-arm64-tests-Python-${{ matrix.python-version }}
-    runs-on: macOS-14
+    name: ${{ matrix.os }}-arm64-tests-Python-${{ matrix.python-version }}
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         # Normally we test min and max version but we can't run python
         # 3.9 on arm64 until actions/setup-python#808 is resolved
-        python-version: ["3.10", "3.13"]
+        python-version: ["3.9", "3.13"]
+        os: ["macOS-14", "ubuntu-24.04-arm"]
```

Comment:
This comment must be out-of-date now?

Reply:
Removed in: https://github.com/Qiskit/qiskit/pull/13737/commits/962594e1de794ea04072d2b47720e1f2953325e6